imjoy/core/services.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout)
+logger = logging.getLogger("imjoy-core")
+logger.setLevel(logging.INFO)
+
+
+class Services:
+    def __init__(self, imjoy_api=None):
+        self._services = []
+        self.imjoy_api = imjoy_api
+
+    def register_service(self, plugin, service):
+        self._services.append(service)
+
+    def get_plugin(self, plugin, config):
+        pass
+
+    def generate_presigned_token(self, plugin):
+        pass
+
+    def get_service(self, plugin, name):
+        return list(filter(lambda x: x.get("name") == name, self._services))[0]
```

Comment:
It's more readable and simpler to use a list comprehension instead of lambda + filter. And since we only need the first item in the list we can use `next` and a generator expression which is faster than creating a whole list.
```suggestion
        return next(
            service for service in self._services if service.get("name") == name
        )
```

Reply:
Nice, didn't know you can use next like that.